monkey/monkey_island/cc/resources/attack_telem.py

```diff
@@ -20,5 +20,5 @@ def post(self, technique):
         :param technique: Technique ID, e.g. T1111
```

Comment:
There's a very critical issue with the attack telem the way it is. I'll write in slack and we'll solve it on another branch once this is merged